templates/CRM/Contact/Form/Contact.tpl

```diff
@@ -78,7 +78,7 @@
 
     {foreach from = $editOptions item = "title" key="name"}
       {if $name eq 'CustomData' }
-        <div id='customData'>{include file="CRM/Contact/Form/Edit/CustomData.tpl"}</div>
+        <div id='customData'>{include file="CRM/Contact/Form/Edit/CustomData.tpl" isSingleRecordEdit=TRUE}</div>
```

Comment:
Smarty include params are case-sensitive and it actually wants lower case otherwise it thinks it's a string. For TRUE that ends up the same, but should this be false?

Reply:
I fixed the casing - just checking re what it should be - it would up that TRUE was right on the contact edit page cos they always are single & it fails on other checks - so this makes that explicit - just gotta check the multi- tab

Reply:
@eileenmcnaughton it should be `false`. If you run it you'll see the custom fields look "wrong" with `true`. They're there and the right labels/types, etc, but they aren't lined up the usual way.

Reply:
OK - fixed